### DISCUSSION TITLE
Video Stream Only Showing Streaks on Cameras Webpage

### DISCUSSION THREAD

**Discussion by husain3** [Motion-Project/motion#1528] [General]
Was kinda wondering if you've encountered this problem before. With one or more of the rtsp streams displayed on the hosted Motion webpage at port 8080 looking like the attached photo.

![Screen Shot 2022-04-02 at 2 24 21 PM](https://user-images.githubusercontent.com/12962124/167465407-5dbdfd76-427a-4cb2-895c-c3ef5946cdfe.png)

**Comment by tosiara** [comment]
Those are camera decompression artifacts. Either network issue (wireless? udp?) or cpu resource issue (low power arm device?). Provide more details on your setup

**Reply by husain3** [reply]
Computer running motion has an i5-6500T. Normally these errors happen when Motion detects movement and starts recording.

**Reply by husain3** [reply]
> Those are camera decompression artifacts. Either network issue (wireless? udp?) or cpu resource issue (low power arm device?). Provide more details on your setup

What else do you need to know?

**Comment by tosiara** [comment]
I haven't got any information I asked for yet. Upload your full log (-d 9) while you reproduce the issue and a corresponding video file
